.github/pull_request_template.md

```diff
@@ -1,66 +1,51 @@
-<!-- Start by describing the changes made in this Pull Request, and the reason for such changes. -->
+<!-- Please do not remove any information from this pull request. Instead, add N/A or leave blank if not applicable -->
 
-<!-- Reference any related issues or PRs here -->
+## What
 
 Fixes #
 
-<!-- Don't forget to update the title with something descriptive. -->
-<!-- If you can, add the appropriate labels -->
+## Why
 
-#### Accessibility
+<!-- Describe the reason for your changes. This will help the reviewer and future readers get additional context -->
 
-<!-- If you've changed or added any interactions, check off the appropriate items below. You can delete any that don't apply. Use this space to elaborate on anything if needed. -->
+## Screenshots or screencast
 
-- [ ] I've tested using only a keyboard (no mouse)
-- [ ] I've tested using a screen reader
-- [ ] All animations respect [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
-- [ ] All text has [at least a 4.5 color contrast with its background](https://webaim.org/resources/contrastchecker/)
-
-#### Other Checks
-
-- [ ] This PR has either a `[type]` label or a `[skip-changelog]` label.
-- [ ] This PR adds/removes a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
-- [ ] This PR adds/removes an experimental interfaces and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
-- [ ] I tagged two reviewers because this PR makes queries to the database or I think it might have some security impact.
-
-### Screenshots
-
-<!-- If your change has a visual component, add a screenshot here. A "before" screenshot would also be helpful. -->
+<!-- Any screenshots of UI changes will be helpful to include here -->
```

Comment:
I think we can update it to:
```
<!-- Any screenshots of UI changes will be helpful to include here. Use "N/A" if not applicable. -->
```

This will act as a reminder that we can't remove this section as well. What do you think?

Reply:
I like the prompt reminder. I've updated it to say "Leave blank" if not applicable instead.